docs/resources/offload.md

```diff
@@ -3,12 +3,16 @@
 page_title: "liqo_offload Resource - liqo"
 subcategory: ""
 description: |-
-  Offload a namespace.
+  Once a given namespace is selected for offloading, Offload resource extends
+  it across the cluster boundaries, through the the automatic creation of
+  twin namespaces in the subset of selected remote clusters.
```

Comment:
```suggestion
  twin namespaces in the selected remote clusters.
```
(I would avoid complex sentences which are definitely more precise, but also more difficult to read and understand).